### DISCUSSION TITLE
Inactive Issues

### DISCUSSION THREAD

**Discussion by coreywillwhat** [audiconnect/audi_connect_ha#329] [General]
I see we have the workflow in place for inactive issues to mark as stale > close, but there are issues that go back quite a ways. Do we want to clean the really old ones up?

**Comment by coreywillwhat** [comment]
Maybe close anything older than 90 days with generic comment:
"This issue is now closed due to a period of inactivity. If the problem persists, we encourage you to open a new issue with updated details and documentation. Thank you for your understanding."

**Comment by cdnninja** [comment]
Write now the workflow will do that.  However it is very long lead times.   I'm open to shortening the lead time or changing the closure message.

**Comment by cdnninja** [comment]
Here is the config.   https://github.com/audiconnect/audi_connect_ha/blob/master/.github/workflows/inactiveIssues.yml

I actually wanted to shorten the thresholds at some point. Just wanted to get it in place to see how it worked first.  

How long should each one be set to?

**Comment by coreywillwhat** [comment]
I'd say 30 days to Stale, then another 14 days to Close?
This would give maintainers a chance to bump posts that still need to be worked on or addressed.

What are you thinking?

**Comment by cdnninja** [comment]
I created PR #336  Let me know if those numbers make sense.
